frontend/src/metabase/lib/urls/questions.ts

```diff
@@ -38,8 +38,11 @@ export function question(
 
   if (query && typeof query === "object") {
     query = extractQueryParams(query)
-      .filter(([key, value]) => value !== undefined)
-      .map(kv => kv.map(encodeURIComponent).join("="))
+      .map(([key, value]) =>
+        value == null
+          ? `${encodeURIComponent(key)}=`
+          : [key, value].map(encodeURIComponent).join("="),
+      )
```

Comment:
Now it's possible for parameter values to be null, we represent them in the URL as empty string values like `?param=`